src/components/VmSnapshots/index.jsx

```diff
@@ -98,14 +99,14 @@ class VmSnapshot extends React.Component {
           </span>
           {snapshotMemoryInfo}
           {
-            !isUp
+            !isUp && !isActive
               ? (<Button bsStyle='default' bsSize='small' onClick={this.handleOpenRestoreDialog} style={{ marginRight: '5px' }}>
                 {msg.restore()}
               </Button>)
               : null
           }
           {
-            showSettings
+            showSettings && !isActive
```

Comment:
Completely agree. Moreover avoiding ternary operator in this case is a good practice already commonly used within the rest of the code.
Anyway, due to time-pressure, I'm approving that since there's no affect on functionality.

@bond95 , can you please fix that in a follow-up?